spec/base_spec_helper_system.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+# This file defines configurations that are universal to all spec types (feature, system, etc)
+
+ENV["RAILS_ENV"] ||= 'test'
+
+require 'simplecov' if ENV["COVERAGE"]
+require 'rubygems'
+require 'pry' unless ENV['CI']
+
+require_relative "../config/environment"
+require 'rspec/rails'
+require 'capybara'
+require 'rspec/retry'
+require 'paper_trail/frameworks/rspec'
+require "paperclip/matchers"
+require "factory_bot_rails"
+
+
+require 'shoulda/matchers'
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
+
+# Allow connections to selenium whilst raising errors when connecting to external sites
+require 'webmock/rspec'
+WebMock.enable!
+WebMock.disable_net_connect!(
+  allow_localhost: true,
+  allow: 'chromedriver.storage.googleapis.com'
+)
+
+# Requires supporting ruby files with custom matchers and macros, etc,
+# in spec/support/ and its subdirectories.
+Dir[Rails.root.join("spec/system/support/**/*.rb")].sort.each { |f| require f }
```

Comment:
Committed after changes: before this line was:
`Dir[Rails.root.join("spec/support/**/*.rb")].sort.each { |f| require f }`